VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py

```diff
@@ -0,0 +1,279 @@
+###########################################################
+# $HeadURL$
+# File: OcciEndpoint.py
+# Author: A.T.
+###########################################################
+
+"""
+   OcciEndpoint is Endpoint base class implementation for the Occi cloud service.
+"""
+
```

Comment:
This is not for OCCI :)

and the line  # $HeadURL$ is not needed.
... and you are not A.T. :)